proposals/lip-0047.md

```diff
@@ -520,7 +520,7 @@ def execute(trs: Transaction) -> None:
     )
 ```
 
-The function `verifyWeightedAggSig` is specified in [LIP 0038][lip-0038], `getValidatorKeys` is exposed by the [Validators module][lip-0044]. The schema `validatorSignatureMessageSchema` is:
+The function `verifyWeightedAggSig` is specified in [LIP 0062][https://github.com/LiskHQ/lips/blob/main/proposals/lip-0062.md#signing-and-verifying-with-bls], `getValidatorKeys` is exposed by the [Validators module][lip-0044]. The schema `validatorSignatureMessageSchema` is:
```

Comment:
This does not show up correctly.
```suggestion
The function `verifyWeightedAggSig` is specified in [LIP 0062](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0062.md#signing-and-verifying-with-bls]), `getValidatorKeys` is exposed by the [Validators module][lip-0044]. The schema `validatorSignatureMessageSchema` is:
```

Alternatively, this link can be included in the end of the file and reference it here with its name (see also my comment on LIP 68)

Reply:
changed. Thanks for carefully checking!